Framework/PythonInterface/plugins/algorithms/FindUBFromScatteringPlane.py

```diff
@@ -0,0 +1,135 @@
+
+        # declaring intake variables to define the scattering plane,
+        self.declareProperty(
+            FloatArrayProperty(name="Vector1", direction=Direction.Input, validator=array_length_3),
+            doc="First vector in the scattering plane in reciprocal lattice units - i.e. H,K,L",
+        )
+        self.declareProperty(
+            FloatArrayProperty(name="Vector2", direction=Direction.Input, validator=array_length_3),
+            doc="Second vector in the scattering plane in reciprocal lattice units - i.e. H,K,L",
+        )
+
+        # declaring intake variables abc, alpha, beta, gamma, and the peak
+        self.declareProperty(
+            IPeaksWorkspaceProperty(name="PeaksWorkspace", defaultValue="", direction=Direction.Input),
+            doc="PeaksWorkspace with 1 peak in the scattering plane, if more than 1 peak is provided"
+            "the first peak in the Workspace is chosen",
+        )
+
+        self.declareProperty(
+            name="a", direction=Direction.Input, validator=lower_lattice_parameter_limit, defaultValue=1.0, doc="Lattice Parameter a"
+        )
+        self.declareProperty(
+            name="b", direction=Direction.Input, validator=lower_lattice_parameter_limit, defaultValue=1.0, doc="Lattice Parameter b"
+        )
+        self.declareProperty(
+            name="c", direction=Direction.Input, validator=lower_lattice_parameter_limit, defaultValue=1.0, doc="Lattice Parameter c"
+        )
+
+        self.declareProperty(
+            name="alpha", direction=Direction.Input, validator=angle_limit, defaultValue=90.0, doc="Lattice Parameter alpha"
+        )
+        self.declareProperty(name="beta", direction=Direction.Input, validator=angle_limit, defaultValue=90.0, doc="Lattice Parameter beta")
+        self.declareProperty(
+            name="gamma", direction=Direction.Input, validator=angle_limit, defaultValue=90.0, doc="Lattice Parameter gamma"
+        )
+
+    def exec_child_alg(self, alg_name: str, **kwargs):
+        alg = self.createChildAlgorithm(alg_name, enableLogging=False)
+        alg.setAlwaysStoreInADS(False)
+        alg.initialize()
+        alg.setProperties(kwargs)
+        alg.execute()
+        out_props = tuple(alg.getProperty(prop).value for prop in alg.outputProperties())
+        return out_props[0] if len(out_props) == 1 else out_props
+
+    # Can change these to be 2 arrays of abc, and alpha,beta,gamma if that is preferred
+    def PyExec(self):
+        # getting lattice parameters
+        a = self.getProperty("a").value
+        b = self.getProperty("b").value
+        c = self.getProperty("c").value
+
+        alpha = self.getProperty("alpha").value
+        beta = self.getProperty("beta").value
+        gamma = self.getProperty("gamma").value
+
+        # getting vectors to define the plane
+        vector_1 = np.array(self.getProperty("Vector1").value)
+        vector_1 = V3D(vector_1[0], vector_1[1], vector_1[2])
```

Comment:
```suggestion
        vector_1 = V3D(*self.getProperty("Vector1").value)
```
and remove line above (same for `vector_2`)